queue_job/jobrunner/runner.py

```diff
@@ -424,6 +450,11 @@ def initialize_databases(self):
                         self.channel_manager.notify(db_name, *job_data)
                 _logger.info("queue job runner ready for db %s", db_name)
 
+    def requeue_dead_jobs(self):
+        for db in self.db_by_name.values():
+            if db.has_queue_job:
+                db.requeue_dead_jobs()
+
```

Comment:
```suggestion
```

Copy paste issue?

Reply:
No, it is a function on class `QueueJobRunner` that will call the function `requeue_dead_jobs` on class `Database`.
What might confuse you is that both these two classes are defined in the same class.

Reply:
Indeed!